### PR TITLE
[RDBMS] `az mysql flexible-server restore/georestore/replica create`: Add public-access parameter for restore, replication and georestore

### PR DESCRIPTION
**Pull request by Ruiion**
**Related command**
az mysql flexible-server restore --public-access
az mysql flexible-server georestore --public-access
az mysql flexible-server replica create --public-access 

**Description**
This PR uses latest Python SDK package which in turn uses the latest GA API version 2022-09-30. This PR will allow customers to enable/disabled public access when creating replica and restore.

**Testing Guide**
Covered by E2E

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yonzhan** [comment]
Please fix CLI issues first.

File: src/azure-cli/setup.py [hunk 1]
```diff
@@ -107,7 +107,7 @@
     'azure-mgmt-netapp~=10.0.0',
     'azure-mgmt-policyinsights~=1.1.0b2',
     'azure-mgmt-privatedns~=1.0.0',
-    'azure-mgmt-rdbms~=10.2.0b7',
+    'azure-mgmt-rdbms~=10.2.0b8',
```
Comment (evelyn-ys) [inline src/azure-cli/setup.py:110]: It's suggested to use `==` instead of `~=` just in case there's new beta version contains breaking change

Comment (honghr) [reply src/azure-cli/setup.py:110]: This is OK, we will update this when the sdk releases a new version.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**